boards/arm/mimxrt1050_evk/doc/index.rst

```diff
@@ -298,6 +298,10 @@ the `OpenSDA J-Link MIMXRT1050-EVK-Hyperflash Firmware`_. Check that jumpers
 J32 and J33 are **on** (they are on by default when boards ship from the
 factory) to ensure SWD signals are connected to the OpenSDA microcontroller.
 
+Follow the instructions in `Enable QSPI flash support in SEGGER JLink`_
+in order to support your EVK if you have modified it to boot from QSPI NOR
+flash as specifiee by NXP AN12108.
```

Comment:
Just a small typo: `specifiee` -> `specified`

Reply:
You are right, thank you. Fixed on my last force push.